components/IdentifyUser.js

```diff
@@ -1,12 +1,55 @@
-import React, {useState} from "react";
+import React, { useEffect, useState } from "react";
 import { StyleSheet, View, Text } from "react-native";
 import { TextInput } from "react-native-gesture-handler";
 import FeatureButton from "./common/FeatureButton";
 import { SubHeaderText, SimpleText } from "./common/Text";
 import { CustomerIO } from "customerio-reactnative";
+import { checkNotifications, requestNotifications, RESULTS } from "react-native-permissions";
 
 const IdentifyUser = (props) => {
 
+    const handlePermissionRequest = (permission, onGranted, onFailed) => {
+        permission
+            .then((result) => {
```

Comment:
This needs to be verified on iOS app as well. We have native code in place in iOS to seek user permissions for notifications and using react-native-permissions could possibly add a duplicate but I would like to add this possibility too in our docs to allow flexibility to customers to seek permissions anytime in their app and not just at the app launch !

Reply:
I agree as there could be some cases where customers might want to explain end users why they need to enable notifications before asking for the permission.

For testing on iOS, I think we should wait for #35 to be merged first, I'll then test this before merging.